deployment/terraform/storage.tf

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
These resources are encapsulated in our [`terraform-aws-s3-origin` module](https://github.com/azavea/terraform-aws-s3-origin)